src/main/cpp/vision/DragonQuest.h

```diff
@@ -74,15 +76,22 @@ class DragonQuest : public DragonDataLogger, public DragonVisionPoseEstimator
     nt::DoubleArrayTopic m_rotationTopic;
     nt::IntegerTopic m_frameCountTopic;
     nt::DoubleArrayPublisher m_initialPosePublisher;
+    nt::DoubleSubscriber m_heartbeatRequestSub;
+    nt::DoublePublisher m_heartbeatResponsePub;
 
     bool m_hasreset = false;
     bool m_isConnected = false;
 
+    frc::Transform2d m_robotToQuestTransform; // <I> Transform from robot center to Quest (used to calculate the quest pose from the robot pose)
     frc::Transform2d m_questTransform;
 
+    frc::Pose2d m_rawQuestPose;
+
     const double m_stdxy = 0.5;
```

Comment:
probably want to make these smaller (encoders example is using 0.01, 0.01, 0.01)